src/OnlineStatistics.jl

```diff
@@ -0,0 +1,27 @@
+
+mutable struct OnlineStatistics{T}
```

Comment:
Do you plan to keep this around?

Reply:
I think it could later be useful in other places, but I removed the `include` from Sunny.jl.